### PR TITLE
Removed boxen

### PR DESCRIPTION
**Pull request by FarazzShaikh**
# Remove boxen (#23)

It's not pretty but, Removed boxen although it still needs [ansi-parser](https://www.npmjs.com/package/ansi-parser). 9 -> 1 dependancies. Might work on removing that in the future. 

Not sure why it still shows 16 of my previous commits. I have been using the library to sync itself with its upstream, maybe this is a bug? 🤔

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -16,10 +16,18 @@ async function syncFork(opts) {
     const upstream = opts.upstream || 'upstream';
     const branch = opts.branch || 'master';
 
+    let conflictResolution = '';
```
Comment (Andrew-Colman) [inline lib/utils.js]: It's including the old code

**Comment by Andrew-Colman** [comment]
clever solution, but adds a new level of complexity 

simple solution to remove all deps:    
(es6 template literals)
```js
const box = `\n
┌────────────┐
│  ${msg}    │
└────────────┘`;
```

as the message will be always the same and same length

**Comment by FarazzShaikh** [comment]
I could've that but won't that break the shape when say the version numbers become 2 digit? For example go from

`0.1.8 -> 0.1.9`

To

`0.1.9 -> 0.1.10`

There's one extra character now

**Review by Andrew-Colman — COMMENTED** [review]
> I could've that but won't that break the shape when say the version numbers become 2 digit? For example go from
> 
> `0.1.8 -> 0.1.9`
> 
> To
> 
> `0.1.9 -> 0.1.10`
> 



> I could've that but won't that break the shape when say the version numbers become 2 digit? For example go from
> 
> `0.1.8 -> 0.1.9`
> 
> To
> 
> `0.1.9 -> 0.1.10`
> 
> There's one extra character now

@FarazzShaikh 
it doesn't seems that will have too many versions 

the next version is 0.6.0

and if it's stable enough for a while, I'll publish it as a 1.0.0 then just fix reported bugs

**Comment by FarazzShaikh** [comment]
@Andrew-Colman fair enough then, I will simply hard code it and add it to the PR

**Pull request closed without merging**